runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -238,6 +265,7 @@ public static String getFullyQualifiedTableName(String namespace, String tableNa
      */
     private <T extends Message> void addTypeToClassMap(T msg) {
         String typeUrl = getTypeUrl(msg.getDescriptorForType());
+        log.error("Adding "+ typeUrl + " serializer map..");
```

Comment:
log.error? Is this used when you were debugging and forgot to delete?

Reply:
Good catch! thanks - this is from the debugging patch.